docs/data-ai/ai/train.md

```diff
@@ -300,6 +541,10 @@ The script you are creating must take the following command line inputs:
 - `dataset_file`: a file containing the data and metadata for the training job
 - `model_output_directory`: the location where the produced model artifacts are saved to
 
+If you used the version with parsed labels, it will also take the following command line inputs:
```

Comment:
```suggestion
If you used the training script template that allows users to pass in lables, it will also take the following command line inputs:
```